app/lib/theme/customTransformErrors.ts

```diff
@@ -0,0 +1,18 @@
+import { AjvError } from "@rjsf/core";
+
+export const customTransformErrors = (
+  errors: AjvError[],
+  customFormatsErrorMessages: { [key: string]: string }
+) => {
+  // Ignore oneOf errors https://github.com/rjsf-team/react-jsonschema-form/issues/1263
+  return errors
+    .filter((error) => error.name !== "oneOf")
+    .map((error) => {
+      if (error.name !== "format") return error;
```

Comment:
Can we update `is a required property` (`error.name == "required"`) to a more actionable phrasing?
e.g. "Please enter a value.". For dropdown and radio buttons, "Please select a value." would be more appropriate, but I'm not sure if we can easily find out the input type here

Reply:
I fiddled with getting the input type into the error object, but didn't find a quick solution. Leaving it as "Please enter a value" for all "required" error messages.